ComprefaceTestApp/Services/ExampleSubjectService.cs

```diff
@@ -51,4 +58,59 @@ public async Task<ListAllExampleSubjectResponse> GetAllExampleSubjects(ListAllEx
 
         return response;
     }
+
+    public async Task<DeleteAllExamplesResponse?> ClearSubjectAsync(DeleteAllExamplesRequest request)
+    {
+        var requestUrl = $"{_httpClient.BaseAddress}recognition/faces";
+
+        var response = await requestUrl.
+            SetQueryParam("subject", request.Subject)
+            .DeleteAsync(HttpCompletionOption.ResponseContentRead);
+
+        return await response.ResponseMessage.Content.ReadFromJsonAsync<DeleteAllExamplesResponse>();
+    }
+
+    public async Task<DeleteImageByIdResponse?> DeleteImageByIdAsync(DeleteImageByIdRequest request)
+    {
+        var requestUrl = $"{_httpClient.BaseAddress}recognition/faces";
+
+        var response = await requestUrl
+            .AppendPathSegment(request.ImageId.ToString())
+            .DeleteAsync(HttpCompletionOption.ResponseContentRead);
+
+        return await response.ResponseMessage.Content.ReadFromJsonAsync<DeleteImageByIdResponse>();
+    }
+
+    public async Task<List<Face>> DeletMultipleExamples(DeleteMultipleExamplesRequest deleteMultipleExamplesRequest)
+    {
+        var requestUrl = $"{_httpClient.BaseAddress}recognition/faces";
+
+        var response = await requestUrl
+            .AppendPathSegment("delete")
+            .SendJsonAsync(HttpMethod.Post, deleteMultipleExamplesRequest.ImageIdList.ToList().Select(x => x.ToString()).ToList());
```

Comment:
you can use `PostJsonAsync`  method here it is more readable and after the call you can use `ReceiveJson` method for deserialization then you will not need to call `ReadFromJsonAsync` explicitly

Reply:
Done, thank you!